### PR TITLE
Use Python shebang in Python Postinstall

### PR DESCRIPTION
**Pull request by zrhoffman**
As @ocket8888 mentions in https://github.com/apache/trafficcontrol/pull/4763#discussion_r577819103 , turning a Python script into a polyglot bash script seems convoluted. This PR moves the logic for finding which Python binary to use to the `postinstall` script and uses a standard `python3` shebang for `_postinstall.py`.

-----------------------------------------------

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

* Make sure
* Run the Python postinstall tests, make sure they pass

## PR submission checklist
- [x] Existing tests are sufficient 
- [x] Old shebang was not mentioned in documentation, no documentation change is necessary
- [ ] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_ops/install/bin/postinstall [hunk 1]
```diff
@@ -25,10 +25,17 @@ for arg in "$@"; do
 	esac
 done
 
+PATH+=:/usr/libexec/
+python_bin="$(command -v {python,platform-python}{3{8,.8,6,.6,},2{7,.7,},} | head -n1)"
```
Comment (ocket8888) [inline traffic_ops/install/bin/postinstall:29]: I realize this is recreating the supported versions from the old shebang, but why not include 3.7?

Comment (zrhoffman) [reply traffic_ops/install/bin/postinstall:29]: No reason not to. We could include 3.9, too

Comment (zrhoffman) [reply traffic_ops/install/bin/postinstall]: Including Python 3.7 and 3.9 in 3f272e6f2f

**Review by ocket8888 — APPROVED** [review]
Tests still pass, proper Python version is properly chosen

**Pull request merged**